c++/src/kj/compat/http-test.c++

```diff
@@ -4003,60 +4003,103 @@ KJ_TEST("HttpServer threw exception") {
   KJ_EXPECT(text.startsWith("HTTP/1.1 500 Internal Server Error"), text);
 }
 
-KJ_TEST("HttpServer bad request") {
-  KJ_HTTP_TEST_SETUP_IO;
-  kj::TimerImpl timer(kj::origin<kj::TimePoint>());
-  auto pipe = KJ_HTTP_TEST_CREATE_2PIPE;
-
-  HttpHeaderTable table;
-  BrokenHttpService service;
-  HttpServer server(timer, table, service);
-
-  auto listenTask = server.listenHttp(kj::mv(pipe.ends[0]));
-
-  static constexpr auto request = "GET / HTTP/1.1\r\nbad request\r\n\r\n"_kj;
-  auto writePromise = pipe.ends[1]->write(request.begin(), request.size());
-  auto response = pipe.ends[1]->readAllText().wait(waitScope);
-  KJ_EXPECT(writePromise.poll(waitScope));
-  writePromise.wait(waitScope);
+KJ_TEST("HttpServer request/response") {
```

Comment:
Nitpick: The new name of this test isn't clear that it's testing error cases. I think the old name was better.

```suggestion
KJ_TEST("HttpServer bad request") {
```